src/actions/guides/http_and_routing/routing_and_params.cr

```diff
@@ -306,10 +306,29 @@ class Guides::HttpAndRouting::RoutingAndParams < GuideAction
     end
     ```
 
-    There are a few caveats to the `memoize` macro. It will not memoize `false`, or `nil` return values.
-    If your method returns a "falsey" value, then it will be ran each time. Another thing to note is
-    you can't memoize a method that takes an argument. This is due to the dynamic nature of the
-    arguments.
+    You can also memoize methods that have a `nil` or `false` return value like when finding a record.
+    As well as the ability to pass arguments to your memoized method.
```

Comment:
The second sentence seems like a continuation of the first but I think it is a separate sentence. Maybe something like:

```suggestion
    Memoized methods can return any value, including`nil` or `false`.
    You can also pass arguments to your memoized method.
```

Reply:
Yeah, I wasn't quite sure how to word that. I think your suggestion is good.